keyboards/yushakobo/ergo68/info.json

```
@@ -0,0 +1,225 @@
+            {"matrix": [4, 6], "x": 95, "y": 64, "flags": 1},
+            {"x": 224, "y": 0, "flags": 8},
+            {"x": 214, "y": 0, "flags": 8},
+            {"x": 203, "y": 0, "flags": 8},
+            {"matrix": [5, 0], "x": 224, "y": 10, "flags": 1},
+            {"matrix": [5, 1], "x": 210, "y": 10, "flags": 4},
+            {"matrix": [5, 2], "x": 196, "y": 7, "flags": 4},
+            {"matrix": [5, 3], "x": 182, "y": 6, "flags": 4},
+            {"matrix": [5, 4], "x": 168, "y": 7, "flags": 4},
+            {"matrix": [5, 5], "x": 154, "y": 9, "flags": 4},
+            {"matrix": [6, 6], "x": 140, "y": 15, "flags": 4},
+            {"matrix": [6, 5], "x": 154, "y": 20, "flags": 4},
+            {"matrix": [6, 4], "x": 168, "y": 19, "flags": 4},
+            {"matrix": [6, 3], "x": 182, "y": 17, "flags": 4},
+            {"matrix": [6, 2], "x": 196, "y": 19, "flags": 4},
+            {"matrix": [6, 1], "x": 210, "y": 22, "flags": 4},
+            {"matrix": [6, 0], "x": 224, "y": 22, "flags": 1},
+            {"matrix": [7, 0], "x": 224, "y": 34, "flags": 1},
+            {"matrix": [7, 1], "x": 210, "y": 34, "flags": 4},
+            {"matrix": [7, 2], "x": 196, "y": 31, "flags": 4},
+            {"matrix": [7, 3], "x": 182, "y": 29, "flags": 4},
+            {"matrix": [7, 4], "x": 168, "y": 31, "flags": 4},
+            {"matrix": [7, 5], "x": 154, "y": 32, "flags": 4},
+            {"matrix": [7, 6], "x": 140, "y": 26, "flags": 4},
+            {"matrix": [8, 6], "x": 140, "y": 38, "flags": 4},
+            {"matrix": [8, 5], "x": 154, "y": 44, "flags": 4},
+            {"matrix": [8, 4], "x": 168, "y": 42, "flags": 4},
+            {"matrix": [8, 3], "x": 182, "y": 41, "flags": 4},
+            {"matrix": [8, 2], "x": 196, "y": 42, "flags": 4},
+            {"matrix": [8, 1], "x": 210, "y": 45, "flags": 4},
+            {"matrix": [8, 0], "x": 224, "y": 45, "flags": 1},
+            {"matrix": [9, 0], "x": 224, "y": 57, "flags": 1},
+            {"matrix": [9, 1], "x": 210, "y": 57, "flags": 1},
+            {"matrix": [9, 2], "x": 196, "y": 54, "flags": 1},
+            {"matrix": [9, 3], "x": 175, "y": 54, "flags": 1},
+            {"matrix": [9, 4], "x": 161, "y": 55, "flags": 1},
+            {"matrix": [9, 5], "x": 144, "y": 58, "flags": 1},
+            {"matrix": [9, 6], "x": 130, "y": 64, "flags": 1}
+        ],
+        "max_brightness": 100,
+        "sleep": true,
+        "split_count": [37, 37],
+        "animations":{
+            "alphas_mods": true,
+            "breathing": true,
+            "cycle_all": true,
+            "rainbow_moving_chevron": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "raindrops": true,
+            "solid_reactive": true,
+            "splash": true,
+            "solid_splash": true,
+        },
+        "default":{
+          "hue": 0,
+          "sat": 255,
+          "val": 50,
+          "speed": 127
```

Should avoid duplication of default values
```suggestion
          "val": 50
```